go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/aws/aws-sdk-go v1.44.84
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/clarketm/json v1.14.1
+	github.com/containers/image/v5 v5.23.1
```

Comment:
what's causing all these go.mod changes, and can we avoid it? Ideally i'd like this pr to not involve any changes to the vendor directory.

Reply:
Added to read the container image digest, to be used as the image hash. Removed as we don't cache container images for now.